tests/test_repairs.py

```
@@ -0,0 +1,1323 @@
+    hass.config_entries.async_reload.assert_awaited_once_with(entry.entry_id)
+    hass.config_entries.async_schedule_reload.assert_called_once_with(entry.entry_id)
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "reload_result",
+    [False, HomeAssistantError("reload failed"), KeyError("entry key")],
+)
+async def test_retry_keeps_issue_when_recovery_reload_fails(
+    monkeypatch: pytest.MonkeyPatch,
+    reload_result: bool | Exception,
```

Ruff (0.15.21)

[warning] Boolean-typed positional argument in function definition

(FBT001)